src/components/layout.module.css

```diff
@@ -1,36 +1,37 @@
 .container {
   margin: auto;
+  padding-top: 4rem;
```

Comment:
This is why your page titles weren't displaying. Your navigation is using position absolute, so it exists outside the content flow. This causes all of your content below it to move up to fill the space, which hides your page title below the navbar.

This is usually solved by adding some top padding/margin depending on your markup.